dataland-csvconverter/src/main/kotlin/org/dataland/csvconverter/csv/CsvUtils.kt

```diff
@@ -128,6 +128,31 @@ object CsvUtils {
             .stripTrailingZeros()
     }
 
+    /**
+     * Tries to parse a decimal value from the CSV file with the expected format
+     * XXX.XXX,XXX
+     */
+    fun Map<String, String>.readCsvDecimalNotAllowingNull(
```

Comment:
This function is - except for one row - identical to `readCsvDecimal`.
Please unify - we don't want to have copy&place

Reply:
I removed the function and throw a null pointer exception when it is called. Thinking about a more precise exception type.

Reply:
ok